cmd/argocd/commands/app_actions.go

```diff
@@ -17,10 +16,17 @@ import (
 	"github.com/argoproj/argo-cd/errors"
 	argocdclient "github.com/argoproj/argo-cd/pkg/apiclient"
 	applicationpkg "github.com/argoproj/argo-cd/pkg/apiclient/application"
-	argoappv1 "github.com/argoproj/argo-cd/pkg/apis/application/v1alpha1"
 	"github.com/argoproj/argo-cd/util"
 )
 
+type DisplayedAction struct {
+	Group    string
+	Kind     string
+	Name     string
+	Action   string
+	Disabled bool
+}
```

Comment:
Not sure if this is the best place for this. It is only used in `NewApplicationResourceActionsListCommand`

Reply:
I think it makes sense to keep it here